test_reporting/kusto/setup.kql

```diff
@@ -196,6 +196,11 @@
 ////////////////////////////////////////////////////////////
 .create table ['RebootTimingData'] ingestion json mapping 'RebootTimingDataMapping' @'['
                                                             '{"column":"ReportId","Properties":{"path":"$.id"}},'
+                                                            '{"column":"TrackingId","Properties":{"path":"$.tracking_id"}},'
+                                                            '{"column":"TestbedName","Properties":{"path":"$[\'hostname\']"}},'
```

Comment:
TesbedName is technially not the same as hostname

Reply:
That's true, I can rename it to `Hostname` instead.